### ISSUE TITLE
Página de inicio Condiciones de uso y Sobre este portal

### ISSUE THREAD

**Issue by jag2kn** [SIB-Colombia/species_checklist_frontend#15]
Página de inicio	"Condiciones de uso y Sobre este portal no muestran lo que deben mostrar. 
Ambas páginas están en GitHub: https://github.com/SIB-Colombia/species_checklist_frontend/issues/8"

**Comment by RicardoOrtizG** [comment]
La sección de "Sobre este Portal" cumple con el contenido que está en GitHub: #8. La sección "Condiciones de Uso" no cumple con el contenido, específicamente en el segmento asociado a ¿Cómo publicar listas?, pues está incompleto.

Portal Listas

![image](https://user-images.githubusercontent.com/15789737/48152511-1e85fc00-e292-11e8-9fab-1455ddb1b589.png)

Contenido

![image](https://user-images.githubusercontent.com/15789737/48152550-2cd41800-e292-11e8-874b-07992b31675d.png)

**Comment by iraidamercedes** [comment]
@RicardoOrtizG ¡está este issue completado?

**Comment by RicardoOrtizG** [comment]
No, no está la información completa, al segmento de cómo publicar listas, le falta una parte del texto. Ver issue #8
